conn.go

```diff
@@ -1354,8 +1355,11 @@ func (c *Conn) ConfirmHandshake() error {
 		input = &block{data: buf.Bytes()}
 	}
 
-	for c.phase != handshakeConfirmed {
-		if err := c.readRecord(recordTypeApplicationData); err != nil {
+	// At this point, earlyDataReader has read all early data and received
+	// the end_of_early_data signal. Expect a Finished message.
+	// Locks held so far: c.confirmMutex, c.in
+	for c.phase != handshakeConfirmed { // c.phase == discardingEarlyData || c.phase == waitingClientFinished
```

Comment:
This comment is helpful, but looks odd after the {

Reply:
Fixed this and the other comment, thanks for reviewing